### PR TITLE
Generate Rails API and add Vagrantfile.

### PR DESCRIPTION
**Pull request by patrickberkeley**

### CODE REVIEW

**Comment by patrickberkeley** [comment]
//cc @alanpeabody

File: config/application.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require File.expand_path('../boot', __FILE__)
+
+require 'rails/all'
```
Comment (alanpeabody) [inline config/application.rb]: You can only require a subset of rails here.

Comment (alanpeabody) [inline config/application.rb]: Something like:

``` ruby
require "active_model/railtie"
require "active_record/railtie"
require "action_controller/railtie"
require "action_mailer/railtie"
```

File: .vagrant/machines/default/virtualbox/index_uuid [hunk 2]
```diff
@@ -0,0 +1 @@
+6519511837264aabb88f939c87d68062
```
Comment (alanpeabody) [inline .vagrant/machines/default/virtualbox/index_uuid]: `.vagrant` should be added to `.gitignore`

File: test/test_helper.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  fixtures :all
+
+  # Add more helper methods to be used by all tests here...
+end
```
Comment (alanpeabody) [inline test/test_helper.rb]: If you plan on using Rspec this whole folder can go. For next time: `rails-api g name -T` skips test unit.

**Comment by patrickberkeley** [comment]
@alanpeabody thanks! Updated based on your feedback.

**Comment by alanpeabody** [comment]
:rowboat: :it:

**Pull request merged**